src/components/templates/ProductTemplate/ProductTemplateCard/ProductTemplateCard.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import { useViewport } from '../../../../hooks/useViewport';
+import Heading from '../../../atoms/Heading/Heading';
+import Text from '../../../atoms/Text/Text';
+import Card from '../../../organisms/Card/Card/Card';
+import { breakpoints } from '../../../../constants/breakpoints';
+import { colors } from '../../../../constants/colors';
+import { maxMedia } from '../../../../helpers/responsiveness';
+
+interface ProductTemplateCardProps {
+  notification?: ReactNode;
+  title?: ReactNode;
+  subtitle?: ReactNode;
```

Comment:
Did we decide against removing these props in favour for just passing in `children`?

Reply:
It's still an open discussion https://github.com/zopaUK/react-components/pull/590#discussion_r490264861